docs/B-Computations/style-guidelines.md

```diff
@@ -17,15 +17,15 @@ After reading this section, you will be able to:
 
 A well-written program is a pleasure to read. The coding style is consistent and clear throughout.  The programmer looking for a bug sees a well-defined structure and finds it easy to focus on the portion of the code that is suspect. The programmer looking to upgrade the code sees how and where to incorporate changes. Although several programmers may have contributed to the code throughout its lifetime, the code itself appears to have been written by one programmer.
 
-This chapter describes the coding style used throughout these notes and recommended for an introductory course in programming. This style is based in part on the Linux kernel coding style published at [https://www.kernel.org/doc/html/latest/process/coding-style.html\#codingstyle](https://www.kernel.org/doc/html/latest/process/coding-style.html#codingstyle). Conventions that apply specifically to syntax introduced in subsequent chapters are described in those chapters. 
+This chapter describes the coding style used throughout these notes and recommended for an introductory course in programming. This style is based in part on the [Linux kernel coding style](https://www.kernel.org/doc/html/latest/process/coding-style.html#codingstyle). Conventions that apply specifically to syntax introduced in subsequent chapters are described in those chapters. 
```

Comment:
Consider adding descriptive texts for the links. It will add a descriptive text for users when they hover over the link.

Reply:
Fixed in my latest commit. Thank you for the feedback :)